org.eclipse.xtext/src/org/eclipse/xtext/XtextRuntimeModule.java

```diff
@@ -122,14 +122,8 @@ public Class<? extends IValueConverterService> bindIValueConverterService() {
 	/**
 	 * @since 2.9
 	 */
-	@SuppressWarnings({"removal","deprecation"})
+	@Deprecated
 	public void configureIXtext2EcorePostProcessor(Binder binder) {
```

Comment:
Likewise.

Reply:
i wanted to keep these in a first step and do their removal in a 2nd stage